queries.md

```diff
@@ -0,0 +1,53 @@
+# Query documentation
+
+## General notes
+
+## Highlights
+
+### Description of query
+
+The highlights query is composed of two parts:
+
+( A) Select **1 item** from History where:
```

Comment:
Not sure if this should be `( A )` (with a space after "A" -- to match `( B )` below), or if we want to tempt fate and make it an ordered list.